### DISCUSSION TITLE
fan detected problem on macs fan control / etc

### DISCUSSION THREAD

**Discussion by iirvanard** [itsmeshibintmz/Acer-Nitro-5-AN515-54-Monterey#13] [Q&A] [answered]
Hello, I want to ask if there is any problem with your fan control? I am experiencing issues with my fans as they are working but I cannot control the speed because it is not being detected.

iam using sonoma 14.5 os 🙏

**Comment by itsmeshibintmz** [comment, marked as answer]
Yes, so far I couldn't figure out fan control. try checking in the discord group. I sold my Nitro so don't have access to try anymore. 
Also how is sonoma working for you, All features are working? any issues other than fan control?

**Reply by iirvanard** [reply]
So far, it's only fan control, thanks for the help.

**Reply by itsmeshibintmz** [reply]
okay, how did u manage to get Sonoma working? I tried before selling mine, but ran into several issues.

**Reply by iirvanard** [reply]
After installation, I encountered some issues, such as WiFi, Bluetooth, etc. However, I just needed to upgrade the kext to the latest version, and it started working. Some kext versions don't work on Sonoma. Additionally, I need to update the configuration, but I forgot what changes I made. and because micron doesnt support hackintosh so i need to disabled it in ssdt

**Reply by itsmeshibintmz** [reply]
Happy for you. Glad you made it work!
